v3/bestway/best.json

```diff
@@ -0,0 +1,305 @@
+{
+    "name": "Best70",
```

Comment:
Rename file to `bestway.json`

The name is Best70 but in QMK it says just `Best`.
Please keep one of them to be consistent